test/addons/worker-addon/test.js

```diff
@@ -0,0 +1,21 @@
+// Flags: --experimental-worker
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const child_process = require('child_process');
+const path = require('path');
+const { Worker } = require('worker_threads');
+const binding = path.resolve(__dirname, `./build/${common.buildType}/binding`);
```

Comment:
Can you add a test when both the main thread and the worker thread require the same addon?